tests/src/Kernel/Form/AuthenticationFormTest.php

```diff
@@ -103,6 +97,65 @@ public function testGenerateNewAuthKey() {
     $this->assertSame(EdgeKeyTypeInterface::EDGE_AUTH_TYPE_BASIC, $decoded['auth_type']);
   }
 
+  /**
+   * Test private file path errors.
+   *
+   * @throws \Drupal\Core\Form\EnforcedResponseException
+   * @throws \Drupal\Core\Form\FormAjaxException
+   */
+  public function testPrivateFileSystemPathErrors() {
+    $form_state = new FormState();
+    $form = \Drupal::formBuilder()->buildForm(AuthenticationForm::class, $form_state);
+    $this->assertInstanceOf(AuthenticationForm::class, $form_state->getFormObject());
+
+    // Private file system is not configured.
+    $this->assertNotEmpty($form['connection_settings']['unconfigurable']['description']);
+    $this->assertTrue($form['actions']['#disabled']);
+    $this->assertEquals('The Drupal private file setting has not been configured.', $form['connection_settings']['unconfigurable']['label']['#value']);
+
+    // Add file_private_path setting, but not the private file dir.
+    $private_directory = DrupalKernel::findSitePath(Request::create('/')) . '/private';
+    $this->setSetting('file_private_path', $private_directory);
+
+    // Rebuild the form.
+    $form_state = new FormState();
+    $form = \Drupal::formBuilder()->buildForm(AuthenticationForm::class, $form_state);
+    $this->assertInstanceOf(AuthenticationForm::class, $form_state->getFormObject());
```

Comment:
Why this check is necessary? We know for sure what the form builder returned, aren't we?

Reply:
This is from the earlier commited code, did not seem like a bad idea. Removed